danieldupriest/reimbursinator
# Finalize report api

Added a "finalize report" api endpoint. Also changed the subject of the email to reflect the status of the report.

Testing
1. Update details in the `.env` file
2. Run `pipenv` commands, and then `python manage.py runserver`
3. Open postman and import the following link with updated endpoints: https://www.getpostman.com/collections/96e39a77bbd42ac92e13
4. Login as frank and create a new report (unfortunately my reports from testing are still in the DB)
5. Test the `Review Report` command and make sure you see the following output: 
`{
    "message": "Request for review is submitted."
}`
Your email subject should display REVIEW - Request #(num).
6. Test `Submit final report` for the following output: `{
    "message": "Final report submitted."
}`
This email should be in the same email chain as REVIEW, but the new subject should have FINAL

Closes #102
